### PR TITLE
[MAIN] update build_wheels action

### PR DESCRIPTION
**Pull request by Moritz-Alexander-Kern**
Update the build_wheels action to use cibuildwheel version 2.13.1.

With this change wheels will also build for Python 3.11.

### CODE REVIEW

**Review by Moritz-Alexander-Kern — COMMENTED** [review]
build wheels

**Review by Moritz-Alexander-Kern — COMMENTED** [review]
build wheels

**Review by Moritz-Alexander-Kern — COMMENTED** [review]
build wheels

File: .github/workflows/build_wheels.yml [hunk 1]
```diff
@@ -24,7 +24,9 @@ jobs:
       - uses: actions/setup-python@v4
 
       - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==2.3.1
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install cibuildwheel==2.13.1
```
Comment (mdenker) [inline .github/workflows/build_wheels.yml:29]: Is the pipe required?
```suggestion
        run:
          python -m pip install --upgrade pip
          python -m pip install cibuildwheel==2.13.1
```

**Review by mdenker — APPROVED** [review]

**Review by Moritz-Alexander-Kern — COMMENTED** [review]
build wheels

**Review by Moritz-Alexander-Kern — COMMENTED** [review]
build wheels

**Pull request merged**